core/shared/src/main/scala-2.13/fs2/CollectorPlatform.scala

```diff
@@ -13,6 +15,20 @@ private[fs2] trait CollectorPlatform { self: Collector.type =>
   implicit def supportsMapFactory[K, V, C[_, _]](f: MapFactory[C]): Collector.Aux[(K, V), C[K, V]] =
     make(Builder.fromMapFactory(f))
 
+  /**
+    * Use `ArraySeq.untagged` to build a `Collector` where a `ClassTag` is not available.
+    */
+  implicit def supportsTaggedArraySeq[A: ClassTag](
+      a: ArraySeq.type
+  ): Collector.Aux[A, ArraySeq[A]] = {
+    val _ = a
+    make(implicitly[ClassTag[A]] match {
+      case ClassTag.Byte =>
+        Builder.byteArray.mapResult(ArraySeq.unsafeWrapArray).asInstanceOf[Builder[A, ArraySeq[A]]]
+      case _ => Builder.taggedArraySeq[A]
```

Comment:
This specialisation is basically copied from [`Collector.supportsArray`](https://github.com/functional-streams-for-scala/fs2/blob/v2.4.2/core/shared/src/main/scala/fs2/Collector.scala#L27).